chainer/link_hooks/spectral_normalization.py

```diff
@@ -250,8 +272,12 @@ def normalize_weight(self, link, *args, **kwargs):
             W = W / sigma
         if not configuration.config.in_recomputing:
             self.v = v
-            if configuration.config.train:
-                link.xp.copyto(getattr(link, vector_name), u)
+            with chainer.using_device(link.device):
+                if configuration.config.train:
+                    if link.xp is chainerx:
```

Comment:
Why don't you add the comment `# TODO(crcrpar): Remove this when chainerx supports copyto`?